### DISCUSSION TITLE
File structure

### DISCUSSION THREAD

**Discussion by koustakken** [bandinopla/weightxreps-client#27] [Ideas]
Now it’s quite difficult to navigate the project. How about building a file structure? 

Example: 
```
- components /
- - header /
- - - header.tsx
- - - header.module.scss
- - footer /
- - - footer .tsx
- - - footer .module.scss
```

Maybe separated application routing from app.js and decomposed another components?

**Comment by bandinopla** [comment]
there are a lot of files, and this would require to rewrite/fix all the new import paths and could lead to unwanted bugs due to the massive file modification needed to do this. It is true that the files are  abit messy.

I would put this as a very low priority task, because of the risk it might bring and low to no benefit to the end user.
